api/groups/urlParams.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/multiversx/mx-chain-proxy-go/common"
 )
 
+// SystemAccountAddressBech is the const for the system account address
+const SystemAccountAddressBech = "erd1lllllllllllllllllllllllllllllllllllllllllllllllllllsckry7t"
```

Comment:
are we sure there aren't other contracts as well? such as the ones that hold 'burn for all' roles

Reply:
the address for the global settings of esdts is different for every shard.